lib/client-api/src/client_api.js

```diff
@@ -4,12 +4,12 @@ import isPlainObject from 'is-plain-object';
 import { logger } from '@storybook/client-logger';
 import addons from '@storybook/addons';
 import Events from '@storybook/core-events';
+import { toId } from '@storybook/router/utils';
```

Comment:
Do we really want to import the router inside the preview iframe?

Reply:
It only imports the utils, not the entire thing.

Yes this is something we might want to be careful with, but creating a separate package seems a bit overkill?

Reply:
I guess. I was wondering if we could do it the other way round instead.

It just means we have to be careful of a few things:

 - that we keep the `utils.js` entry point in `router` in the published package
 - we don't accidentally import the wrong thing in `utils` down the track (super easy to do and hard to police)

Reply:
If we're really concerned about it, then I suggest we create a package called `@storybook/core-utils`.

Still hard to ensure not 'wrong things' get put into that module, so unless someone has a suggestion on how to actually do it, I'm unsure if we're gaining anything.

I think having the ID generation here is fine, and makes sense since, the primary use-case of the ID is so it can appear in the url and work as a quick lookup; hence the router package is an 'ok' fit.

It's not really an external api, so we can move it around later, if there's trouble.

Reply:
OK, sure. Probably another argument why re-exporting it from core makes sense, IMO. That's the place preview-side consumers (like chromatic) should grab it.

Reply:
or might be called `@storybook/commons`